x/posts/keeper/external_hooks.go

```diff
@@ -10,13 +10,14 @@ import (
 
 // Hooks represents a wrapper struct
 type Hooks struct {
-	k Keeper
+	// Keeper must be the same reference as the original
+	k *Keeper
 }
```

Comment:
The keeper of Hooks must be the same reference as the original, or it occurs following error that the posts keeper of `SubspacesHooks` does not include the relationships `PostsHooks` since they are different the reference:
```
pk := &postskeeper.Keeper{}
sk := &subspaceskeeper.Keeper{}
rk := reactionskeeper.Keeper{}sk.SetHooks(pk.Hooks())
sk.SetHooks(pk.Hooks())
pk.SetHooks(rk.Hooks())
```